doc/docs/concepts.md

```diff
@@ -0,0 +1,123 @@
+
+* The Pipelines Controller is built on top of pipelines and shares many of the same settings
+* Using the controller module you can schedule periodic incremental updates or use a [web control panel](link_to_section_in_adv_guide) to start the pipeline manually
+
+## Flexible and scalable deployment approaches
+
+There are a number of different deployment approaches - see table below. 
+
+Choosing the right approach, comes down to evaluation of a number of factors including the specific use case, requirements of the project, future scalability needs and capabilities of the team. 
+
+| Scenario | Approach | Considerations |
+| -------- | ----------| -------------- |
+|  Simple relational database to power dashboards or reporting | Custom schema defined as ViewDefinition Resources - [see tutorial](tutorial_lossy_example/) | By design this will provide a constrained set of variables in the views |
+| Need for a horizontally scalable architecture | Parquet based DWH and distributed query engine (e.g. SparkSQL) - [see tutorial](tutorial_add_dashboard/) | Will need to manage both distributed storage (Parquet files) and a distributed query engine |
+| Parquet as DWH with a non-distributed OLAP database engine | Parquet based DWH with non-distributed query engine set-up (e.g. SparkSQL or DuckDB) | Scalability and performance (e.g. SparkSQL >> DuckDB) |
+| Exploratory data science or ML use cases | Use the generated Parquet files which as _"near lossless"_ for enhanced data science workflows | Can either use the Parquet or custom schema to power dashboards or reports |
+| Push FHIR data to a central FHIR Data Store for a Shared Health Record or single repository for analytics workloads | Use the Pipelines Controller to push from a FHIR source to a FHIR sink | Management of the intermediate Parquet files created as part of the pipelines | 
+
+## Simplify querying with Pre-defined views
+The heavily nested nature of FHIR resources requires complex SQL queries that can make it difficult to work with for analytics use cases. A common approach to address this is to flatten the data into a set of views (virtual or materialized) which can then be queried using simpler SQL statements. 
+
+FHIR Data Pipes provides two approaches for flattening the FHIR Resources into virtual or materialized views
+
+1.  SQL queries to generate virtual views (outside the pipeline)
+2.  FHIR ViewDefinition resources to generated materialized views (within the pipeline)
+
+For both of these approaches, a set of **"predefined views”** for common resources are provided. These can be modified or extended.
+
+The currently supported list (as at 1st June, 2024), which can be found in the `docker/config/views' directory are:
+
+```
+Condition, DiagnosticReport, Encounter, Immunization, Location, Medicationrequest, Observation, Organization, Patient, Practitioner, PractitionerRole, Procedure
```

Comment:
It is also probably better to remove "```" and make this an itemized list with links to FHIR resource pages for each.
```suggestion
Condition, DiagnosticReport, Encounter, Immunization, Location, Medicationrequest, Observation, Organization, Patient, Practitioner, PractitionerRole, Procedure
```
```suggestion
Condition
DiagnosticReport
Encounter
Immunization
Location
Medicationrequest
Observation
Organization
Patient
Practitioner
PractitionerRole
Procedure
```